### DISCUSSION TITLE
符号对照表标题与列表之间的间距过大

### DISCUSSION THREAD

**Discussion by Witft** [note286/xduts#84] [Q&A] [answered]
设定 style / customize-los = false 时，符号对照表中，标题与“符号、符号名称”这一行的间距看起来比word模板中的要大。模板中的要求是“符号对照表标题字体为黑体，字号为三号，居中排列，行距为固定值20磅，段落间距为段前24磅，段后18磅；正文内容字体为宋体，字号为小四号，行距为固定值20磅”。请问这里的间距需要什么额外设置吗？
![image](https://user-images.githubusercontent.com/44012353/226532952-e0c07c8c-94b5-4b8b-96d8-42f0a7a7a481.png)
![image](https://user-images.githubusercontent.com/44012353/226533251-f6a5644d-b3b3-4638-a20e-cf5761e01058.png)

**Comment by sikouhjw** [comment]
别管这些意义不大的地方，我相信不影响格式审查

**Reply by Witft** [reply]
哈哈，谢谢。主要还是以防万一吧，毕竟学校的模板里已经规定了，要是没写明还好说

**Comment by note286** [comment, marked as answer]
如果你很介意，就自行绘制表格。

**Reply by Witft** [reply]
好的，谢谢。只是自行绘制时有一个使用 longtblr 没法去除续表文本的问题 https://github.com/note286/xduts/discussions/82 这个问题已经回答过了。那我还是保持这样吧

**Reply by note286** [reply]
我现在完全没有时间对代码进行审查，只能做一些回复性工作，你们提的问题我都一一记录下来了，等闲下来了我都会处理的，只能让下一届受益了。

**Reply by Witft** [reply]
好的，非常感谢

**Reply by WXiaoyao** [reply]
您好，我正在自己绘制符号表的表格。但现在发现自定义表格时，字体大小和行间距也需要自己设定，我看官方要求是字体为小四号，行间距为20磅，请问如果我使用命令“\fontsize{12pt}{20pt}\selectfont”，将字体大小设置为12pt（小四号），表格行间距设置为20pt（20磅），这样设置对吗

**Reply by note286** [reply]
@WXiaoyao 我对传统表格了解的不多，据我所致，貌似需要修改`\arraystretch`近似来处理，你自己测试一下吧，直接看PDF效果也行，西安电子科技大学格式审查没那么严格，美观即可，你直接看PDF出来的效果就行。

**Reply by WXiaoyao** [reply]
主要因为后天我们学院要盲审送审了，包括今天我有同学把论文给老师看的时候就被指出来符号表和缩略表的字体大小不对（我们是自定义的，因为使用false的时候标题和列表之间间距过大就暂时没用），所以如果可以的话，能麻烦您做一个示例代码给我们吗？我们几个同学弄这个弄了好几天了也没弄出来，挺着急的...我们希望能做出来类似下图的效果（下图唯一存在的问题就是字体大小和行间距不对）：
<img width="394" alt="1679741583194" src="https://user-images.githubusercontent.com/28039355/227712884-34aa9aa2-6283-4c8d-abae-cf8b5421482a.png">

**Reply by pkeke** [reply]
你好，请问一下你的缩略表的配置整好了嘛

**Comment by ahaaqiya** [comment]
请问我想设置一下对照表的行距应该怎么做呢

**Reply by note286** [reply]
自己手动画表格吧，各凭本事了，或者段落用盒子。我最近没时间维护了。

**Reply by ahaaqiya** [reply]
那请问符号对照表这几个字有办法调整吗，我看手册里没有写

**Reply by note286** [reply]
这你也要调整？你想改成啥？

**Reply by ahaaqiya** [reply]
发现跟往届的字体有点不一样，放到adobe里面查看了一下不是黑体呢，虽然差不多，但是想看看能不能换
![image](https://user-images.githubusercontent.com/91261133/229777302-657278c1-381a-4e0e-ae58-59a864ff61c7.png)

**Reply by note286** [reply]
额，你这描述，你应该问字体如何更改，而不是这几个字如何更改，你可以自行修改中文字体，例如`win`。

**Reply by ahaaqiya** [reply]
这样吗，那是我说的有问题了，，，，我以为字体修改只能针对正文内容，缩略表这边作用不到呢
